pkg/rtc/transportmanager.go

```diff
@@ -424,6 +425,19 @@ func (t *TransportManager) HandleAnswer(answer webrtc.SessionDescription) {
 
 // AddICECandidate adds candidates for remote peer
 func (t *TransportManager) AddICECandidate(candidate webrtc.ICECandidateInit, target livekit.SignalTarget) {
+	if !t.params.Config.UseMDNS {
```

Comment:
nit: should this be in the Transport class instead?

Reply:
Both positions are ok, but filtering it out here can avoid it goes into the transport's event loop

Reply:
sounds good 👍

Reply:
I am late to this. If this does not happen too much, I think it is okay to have put it in transport queue and process it there. Would be nice to have all that encapsulated in that one module.